src/_gettsim/transfers/rente.py

```diff
@@ -792,14 +940,17 @@ def ges_rente_vorauss_besond_langj(ges_rente_wartezeit_45: float) -> bool:
     Eligibility as bool.
 
     """
-    out = ges_rente_wartezeit_45 >= 45
+    out = ges_rente_wartezeit_45
```

Comment:
```suggestion
    return ges_rente_wartezeit_45
```

In case the function stays what it is.

Reply:
changed this